pkg/controller/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -315,3 +325,43 @@ func (r *ReconcileOCSInitialization) Reconcile(request reconcile.Request) (recon
 
 	return reconcile.Result{}, err
 }
+
+// returns a ConfigMap with default settings for rook-ceph operator
+func newRookCephOperatorConfig(namespace string) *corev1.ConfigMap {
+	var defaultCSIToleration string = `
+- key: node.ocs.openshift.io/storage
+  operator: Equal
+  value: "true"
+  effect: NoSchedule`
+
+	config := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      rookCephOperatorConfigName,
+			Namespace: namespace,
+		},
+	}
+	data := make(map[string]string)
+	data["CSI_PROVISIONER_NODE_AFFINITY"] = "cluster.ocs.openshift.io/openshift-storage="
```

Comment:
Shouldn't this be generated from `NodeAffinityKey`?

Reply:
fixed.